pages/about-coq.html

```diff
@@ -78,9 +78,9 @@
     synthetic <a href="/who-did-what-in-coq">who did what</a>
     table).</p>
 
-    <p>The development is coordinated by the ADT Coq (Action for Technological Development), that gathers the teams involved in the implementation of the Coq Proof Assistant. The teams registered in the ADT are the <a href="http://www.inria.fr">INRIA</a> projects <a href="http://www.pps.univ-paris-diderot.fr/pi.r2/">πr²</a> and <a href="http://www-sop.inria.fr/marelle/">Marelle</a>, and the team <a href="http://cedric.cnam.fr/AfficheEquipe.php?id=7&amp;lang=fr">CPR</a> from <a href="http://www.cnam.fr">CNAM</a>.
-</p>
-
+    <p>The development is currently coordinated by a core development
+    team made of Inria researchers and engineers in various
+    project-teams in France.</p>
```

Comment:
```suggestion
    <p>Development is now coordinated by a core development
    team of Inria researchers and engineers in various
    project teams in France.</p>
```

Reply:
I believe "project-team" is a french thing. I'd zap "in various ... in France.".

Reply:
I would leave away any temporal reference like "currently" or "now".

Reply:
Also, is it appropriate to define the core team as Inria personnel only?  Could we be inclusive?  Perhaps just say "a core development team, many of whom are Inria researchers and engineers".  Some developers who are very active are not in France.

Reply:
What distinguishes core team members from other developers?  Is there a secret handshake?

Reply:
I'll remove the paragraph entirely. It was an attempt to update this completely outdated paragraph, but the new one actually doesn't bring anything useful.